tests/monitors/kubernetes_volumes/kubernetes_volumes_test.py

```diff
@@ -0,0 +1,28 @@
+import os
+import pytest
+
+from tests.kubernetes.utils import (
+    run_k8s_monitors_test,
+    get_metrics_from_doc,
+    get_dims_from_doc,
+)
+
+pytestmark = [pytest.mark.kubernetes_volumes, pytest.mark.monitor_without_endpoints]
+
+
+@pytest.mark.k8s
+@pytest.mark.kubernetes
+def test_kubernetes_volumes_in_k8s(agent_image, minikube, k8s_test_timeout, k8s_namespace):
+    monitors = [
+        {"type": "kubernetes-volumes",
+         "kubeletAPI": {"skipVerify": True, "authType": "serviceAccount"}}
+    ]
+    run_k8s_monitors_test(
+        agent_image,
+        minikube,
+        monitors,
+        namespace=k8s_namespace,
+        expected_metrics=get_metrics_from_doc("kubernetes-volumes.md"),
```

Comment:
I'm pretty wary of getting metrics from the docs since it adds a hidden and somewhat unexpected dependency between our doc generation output and the integration tests, which really shouldn't be linked.  Look at the `selfdescribe` output instead since the docs are derived from that.  We can commit the output of the selfdecrribe process to the repo if it will make it easier for these tests to access them.